modules/flowable-cmmn-engine/src/main/java/org/flowable/cmmn/engine/impl/util/JobUtil.java

```diff
@@ -62,19 +63,23 @@ protected static JobEntity createJob(VariableContainer variableContainer, BaseEl
             job.setElementName(((CaseElement) baseElement).getName());
         }
 
-        List<ExtensionElement> jobCategoryElements = baseElement.getExtensionElements().get("jobCategory");
-        if (jobCategoryElements != null && jobCategoryElements.size() > 0) {
-            ExtensionElement jobCategoryElement = jobCategoryElements.get(0);
-            if (StringUtils.isNotEmpty(jobCategoryElement.getElementText())) {
-                Expression categoryExpression = cmmnEngineConfiguration.getExpressionManager().createExpression(jobCategoryElement.getElementText());
-                Object categoryValue = categoryExpression.getValue(variableContainer);
-                if (categoryValue != null) {
-                    job.setCategory(categoryValue.toString());
+        if(CollectionUtils.isEmpty(cmmnEngineConfiguration.getEnabledJobCategories())){
```

Comment:
```suggestion
        if (CollectionUtils.isEmpty(cmmnEngineConfiguration.getEnabledJobCategories())) {
```
The project standards are to use spaces